app/model/editions/EditionsFront.scala

```diff
@@ -18,16 +18,16 @@ case class EditionsFrontMetadata(nameOverride: Option[String], swatch: Option[Sw
 }
 
 case class EditionsFront(
-    id: String,
-    displayName: String,
-    index: Int,
-    canRename: Boolean,
-    isHidden: Boolean,
-    updatedOn: Option[Long],
-    updatedBy: Option[String],
-    updatedEmail: Option[String],
-    metadata: Option[EditionsFrontMetadata],
-    collections: List[EditionsCollection]
+                          id: String,
```

Comment:
`                                  `

![img](https://media.giphy.com/media/l3q2RauzE5Vzf7iYo/giphy.gif)

Reply:
Oh FFS. I thought I fixed that.